build/post-app.js

```diff
@@ -9,22 +10,25 @@ exports.execute = function(options, webapp) {
   // Updates manifest.webapp
   require('./post-manifest').execute(options, webapp);
 
-  require('./multilocale').execute(options, webapp);
+  require('./multilocale').execute(options, webapp).then(function() {
```

Comment:
I'm not sure how much that would change and I'd prefer to not have to rely on trick if I don't have to. We still do wait for all processes in app.js, so not sure if we need to additionally do the same in one of it's subtasks.